.editorconfig

```diff
@@ -1,20 +0,0 @@
-root=true
-
-[*.js, *.ts, *.jsx, *.tsx]
-charset = utf-8
-
-end_of_line = lf
-trim_trailing_whitespace = true
-
-indent_style = space
-indent_size = 2
-
-insert_final_newline = true
-
-[*.yml]
-indent_style = space
-indent_size = 2
-trim_trailing_whitespace = true
-
-[*.md]
-trim_trailing_whitespace = false
```

Comment:
I don't use this - so I don't particularly care about it - but why remove?

Reply:
I can re-add it, but usually editor configs should be git ignored I feel - they are not project related, only user preference right?

Reply:
🤷🏻‍♂️
If it helps people contribute I think it’s fine. Like I said, I don’t use this, so I don’t particularly care :)